components/AcceptPayments/InlineCheckout.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
```

Comment:
I think most of the content here should have been written as mdx instead.

Reply:
Alright, I'll be on that.